Makefile

```diff
@@ -0,0 +1,51 @@
+NAMESPACE := $(shell python -c 'import yaml; print(yaml.safe_load(open("galaxy.yml"))["namespace"])')
+NAME := $(shell python -c 'import yaml; print(yaml.safe_load(open("galaxy.yml"))["name"])')
+VERSION := $(shell python -c 'import yaml; print(yaml.safe_load(open("galaxy.yml"))["version"])')
+MANIFEST := build/collections/ansible_collections/$(NAMESPACE)/$(NAME)/MANIFEST.json
+
+ROLES := $(wildcard roles/*)
+PLUGIN_TYPES := $(filter-out __%,$(notdir $(wildcard plugins/*)))
+METADATA := galaxy.yml LICENSE README.md
+$(foreach PLUGIN_TYPE,$(PLUGIN_TYPES),$(eval _$(PLUGIN_TYPE) := $(filter-out %__init__.py,$(wildcard plugins/$(PLUGIN_TYPE)/*.py))))
+DEPENDENCIES := $(METADATA) $(foreach PLUGIN_TYPE,$(PLUGIN_TYPES),$(_$(PLUGIN_TYPE))) $(foreach ROLE,$(ROLES),$(wildcard $(ROLE)/*/*))
+
+PYTHON_VERSION = $(shell python -c 'import sys; print("{}.{}".format(sys.version_info.major, sys.version_info.minor))')
+
+TOX_ENV ?= py37-ansible2.9-release-static
+
+default: help
+help:
+	@echo "Please use \`make <target>' where <target> is one of:"
+	@echo "  help           to show this message"
+	@echo "  info           to show infos about the collection"
+	@echo "  test           to run molecule tests"
+	@echo "  dist           to build the collection artifact"
+
+info:
+	@echo "Building collection $(NAMESPACE)-$(NAME)-$(VERSION)"
+	@echo "  roles:\n $(foreach ROLE,$(notdir $(ROLES)),   - $(ROLE)\n)"
+	@echo " $(foreach PLUGIN_TYPE,$(PLUGIN_TYPES), $(PLUGIN_TYPE):\n $(foreach PLUGIN,$(basename $(notdir $(_$(PLUGIN_TYPE)))),   - $(PLUGIN)\n)\n)"
+
+test: $(MANIFEST)
+	tox -e $(TOX_ENV)
```

Comment:
So in order to run molecule locally, I have to build and then run a test command?

Reply:
You only need to run `make test`. The Makefile cares for the rest.

Reply:
I think you mentioned this verbally. In order for me to run freeform molecule commands, I have to run `make install && molecule ...`. Correct?

Reply:
Yes, but beware, this is the wrong pr to discuss those things. This one ought to only introduce the repr filter, and shall be rebased once #387 is merged. I think the relevant parts here are already outdated.